cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java

```diff
@@ -391,28 +379,12 @@ public ProgramLiveInfo getLiveInfo(Id.Program program, ProgramType type) {
    */
   private class ClusterResourceReporter extends AbstractResourceReporter {
     private static final String RM_CLUSTER_METRICS_PATH = "/ws/v1/cluster/metrics";
-    private final Path hbasePath;
```

Comment:
if this stuff is unused, we should fix it on develop as well. obviously not this PR.